src/addon-utils.ts

```diff
@@ -82,13 +82,12 @@ function validateObject(
  * @returns {string|null} Error string, or null if no error.
  */
 function validateManifestJson(manifest: Record<string, unknown>): string | null {
-  const manifestTemplate = {
+  const manifestTemplate: Record<string, any> = {
```

Comment:
```suggestion
  const manifestTemplate: Record<string, unknown> = {
```
This would make the linter happy